### PR TITLE
[bitnami/tomcat] fix extraPodSpec bad indentation

### PR DESCRIPTION
**Pull request by itscaro**
### Description of the change

This field is buggy as it is now. Space-trimmed too much which makes the first line of the block to go into volumes section

### Applicable issues

  - fixes #

### Checklist

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [ ] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [x] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/tomcat/Chart.yaml [hunk 1]
```diff
@@ -26,4 +26,4 @@ name: tomcat
 sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/tomcat
   - http://tomcat.apache.org
-version: 10.4.3
+version: 10.4.4
```
Comment (corico44) [inline bitnami/tomcat/Chart.yaml:29]: There is a version conflict. Updates the version taking into account the current version of master.

**Review by corico44 — CHANGES_REQUESTED** [review]
Please, @itscaro , add more relevant info to the PR text.

**Comment by itscaro** [comment]
@corico44 Added. thanks in advance

**Review by corico44 — APPROVED** [review]
LGTM!

**Pull request merged**